src/environment/stella_environment.cpp

```diff
@@ -195,6 +203,26 @@ bool StellaEnvironment::isTerminal() const {
      m_state.getEpisodeFrameNumber() >= m_max_num_frames_per_episode));
 }
 
+void StellaEnvironment::pressSelect(size_t num_steps) {
+  Event* event = m_osystem->event();
```

Comment:
Pass the Event* directly to m_state.pressSelect()

Reply:
I believe I did what you want.